xrviz/utils.py

```diff
@@ -65,9 +65,18 @@ def callback(*events):
 
 
 def is_float(a):
-    # int, 'Nan' are also float
+    # int, 'Nan', 'inf' and '-inf' are also float
     try:
         float(a)
         return True
     except:
         return False
+
+
+def cartopy_geoviews_installed():
```

Comment:
It is somewhat dangerous to repeat code for the purposes of testing. If the test above breaks, it may not be obvious if the error is in the main code or here.

Reply:
Changed. Now compatibility module handles these optional imports.